src/main/webapp/src/app/login-signup-page/login-signup.component.ts

```diff
@@ -68,7 +71,14 @@ export class LoginComponent {
       preferredName: this.user.name,
       biography: this.bio,
       zipCode: this.zipCode,
+      imgSrc: this.user.photoUrl,
     };
+
+    if (this.choice === 'create') {
+      this.admin = true;
+    } else if (this.choice === 'join') {
+      this.done = true;
```

Comment:
Why does choosing `join` imply `done`? Optional: It could be helpful to change the variable names to be more intuitive.